stage1_skim/run/main.go

```diff
@@ -472,12 +466,14 @@ func stage1(rp *stage1commontypes.RuntimePod) int {
 
 				env = append(env, e.Name+"="+containerPath)
 			} else if e.Name == "LD_LIBRARY_PATH" {
+                /*
 				foundLib = true
 				for _, l := range strings.Split(e.Value, ":") {
 					containerLibPath += execDir + workDir + l + ":"
 				}
 
 				env = append(env, e.Name+"="+containerLibPath)
+                */
```

Comment:
Right now, the assumption is that we'll be relying on the system provided libraries, but would like to take a look at $ORIGIN for a later phase.